MPF.Frontend/ViewModels/MainViewModel.cs

```diff
@@ -663,8 +663,7 @@ private void PopulateDrives()
                 CopyProtectScanButtonEnabled = true;
 
                 // Get the current system type
-                if (index != -1)
```

Comment:
This check was to establish that the drive at least attempted to exist. I'd prefer if it stayed.

Reply:
If it doesn't then I cannot do any form of 3DO check or Mac fallback

Reply:
That's unfortunate

Reply:
It tries to read a byte, and if its unreadable (aka no disc actually present), then it will default to no system as is current functionality.
I've also put reading behind a try catch